MAINTAINERS.md

```diff
@@ -141,7 +141,27 @@ the "General" channel of the [Urbit Community group][urbit-community].
 
 This instructions are performed **on release day** to prepare the next release candidate.
 
-- [ ] Merge `master` into `develop` and `git push origin develop`
-  - `master` now has the contents of `release`, which may have deviated from `develop` were any patches or reversions made
-- [ ] Merge `develop` into `release` and `git push origin release`
-- [ ] Update the `VERSION` file to the next version on the `develop` branch and `git push origin develop`
+- [ ] Merge `master` into `develop` (to ensure any changes made to `master` are reflected back into `develop`)
+  - [ ] `git fetch --all`
+  - [ ] `git checkout master`
+  - [ ] `git reset --hard origin/master`
+  - [ ] `git checkout -b merge-master-v{version}-into-develop`
+  - [ ] `git push origin merge-master-v{version}-into-develop`
```

Comment:
Does this work? Assuming `origin/merge-master-v{version}-into-develop` doesn't already exist, I would think you'd need to specify `-u`/`--set-upstream` here.

Reply:
I think you're right @mcevoypeter. We'll test these instructions when we release today and update accordingly.